docs/administer/Idp_groups_integration.md

```diff
@@ -0,0 +1,142 @@
+
+## OKTA
+
+Once you successfully establish an Identity Provider (IdP) [integration with Okta](Idp_integration.md#configure-oidc-on-the-provider-side), the `issuerURL` in this configuration identifies your API server, serving as a unique identifier for your server within Okta. Additionally, you will have a client application, referred to as the Application integration, which is represented by the `clientID`. This setup allows for secure communication and authentication between your application and Okta's services.
+
+To use IdP groups in Percona Everest RBAC, set up the groups claim in your IdP provider settings. Here are the steps to configure the groups claim:
+{.power-number}
+
+1. Navigate to **Security > API**.
+
+2. In the list, find the API server that you use and click **Edit**.
+
+3. Click **Scopes > Add Scope**. 
+
+    !!! note
+        A scope specifies the level and type of access a client application can request from an Identity Provider (IdP) during the authentication process.
+      
+    Add a scope called **groups** while keeping all other options to their default settings.
+
+    ![!image](../images/scr_scope.png)
+
+4. Click **Claims > Add Claim**. 
+
+    !!! note
+        A claim specifies the scope to activate and the information required within that scope.
+
+    1. Add a claim named **groups**.
+
+    2. Set the **Include in token type** to **ID Token** and choose **Always**.
+
+    3. Change the **Value type** to **Groups**.
+
+    4. Select the **Filter** as **Matches regexp** and add a filter to match the desired Okta groups for Everest, for example: .*.
+
+    5. Set **Include in** to **groups**, which is the scope you created earlier.
+
+    ![!image](../images/scr_claim.png)
+
+5. Click **Create**.
+
+6. Click **Access Policies > Add Policy**. A policy restricts how this authorization server is used.
+
+    1. Add **Name** and **Description**.
+
+    2. Assign the policy to the client (Application Integration) you created earlier. The field should auto-complete as you type. Create the policy.
+
+    3. Create the policy.
+
+    ![!image](../images/scr_policy.png)
+
+6. Add a rule to the policy. A Rule defines the details of usage:
+
+    1. Add a name. If you prefer, you can use **default** option.
+
+    2. Fine-tune the settings to align with your organization's security policy. Here are some suggestions:
+
+        - Uncheck all grant types except for the **Authorization Code**.
+        -  Adjust the **token lifetime** to determine how long a session can remain active.
+        - Restrict the refresh **token lifetime** or consider deactivating it completely.
+
+    ![!image](../images/scr_rule.png)
+
+7. Click **Create Rule**.
+
+
+## Configure Group claims in OKTA
```

Comment:
This is actually configuring everest to fetch the groups from the IdP

Reply:
Fixed.